### PR TITLE
Serve legacy notifications

### PR DESCRIPTION
**Pull request by maxov**
First nail in the coffin for non-replyable user threads/message threads. Stops serving non-replyable user threads (next PR will be last nail in the coffin, getting rid of them)

### CODE REVIEW

**Comment by maxov** [comment]
@stkem 
@andrewconner 
@leogrim 
@ryanpbrewster

File: kifi-backend/modules/eliza/app/com/keepit/notify/LegacyNotificationCheck.scala [hunk 1]
```diff
@@ -36,11 +36,7 @@ class LegacyNotificationCheck @Inject() (
     recipient match {
       case u @ UserRecipient(id, experimentEnabled) => experimentEnabled match {
         case None =>
-          shoeboxServiceClient.getUserExperiments(id).map { experiments =>
-            val enabled = experiments.contains(UserExperimentType.NEW_NOTIFS_SYSTEM) || experiments.contains(UserExperimentType.ADMIN)
-            val enabledFlipped = if (flipResults) !enabled else enabled
-            LegacyNotificationCheck.Result(enabledFlipped, u.copy(experimentEnabled = Some(enabledFlipped)))
-          }
+          Future.successful(LegacyNotificationCheck.Result(true, u.copy(experimentEnabled = Some(true))))
```
Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/notify/LegacyNotificationCheck.scala:39]: This is part of your other PR. Should it actually be in here?

Comment (maxov) [inline kifi-backend/modules/eliza/app/com/keepit/notify/LegacyNotificationCheck.scala:39]: This PR depends on the other one, no way to show the diff between them

Comment (ryanpbrewster) [inline kifi-backend/modules/eliza/app/com/keepit/notify/LegacyNotificationCheck.scala:39]: Ah, I see. :+1:

**Comment by stkem** [comment]
:+1:

**Pull request merged**